service/resource/vaultcrt/create.go

```diff
@@ -12,7 +12,33 @@ import (
 	"github.com/giantswarm/cert-operator/service/key"
 )
 
-func (r *Resource) GetCreateState(ctx context.Context, obj, currentState, desiredState interface{}) (interface{}, error) {
+func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange interface{}) error {
```

Comment:
* renamed `Process*State` -> `Apply*Change` and moved up
* renamed `Get*State` -> `new*Change` and moved down